pkg/services/accesscontrol/eval/eval.go

```diff
@@ -0,0 +1,8 @@
+package eval
+
+type Evaluator interface {
+	// Evaluate permissions that are grouped by action
+	Evaluate(permissions map[string]map[string]struct{}) (bool, error)
```

Comment:
Again, just suggesting as it took me a bit of time to wrap my head around what we were evaluating against what:
```suggestion
	// Evaluate check that user permissions encompasses or matches the evaluator's permissions
	Evaluate(userPermissions map[string]map[string]struct{}) (bool, error)
```